library/ssl_misc.h

```diff
@@ -1995,4 +1997,57 @@ int mbedtls_ecp_tls_13_write_group( const mbedtls_ecp_group *grp,
 #endif /* MBEDTLS_ECP_C */
 #endif /* MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL */
 
+/**
+ * To let TLS1.2 and TLS1.3 work together, we divide the task into two steps.
+ * Below inline functions are required by first step.
+ * 1. Configured as TLS1.2 or TLS1.3. Identify the protocol version based on configuration field.
+ * 2. Configured as TLS1.2 and TLS1.3. Identify the protocal version based on runtime parameter.
+ */
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3_EXPERIMENTAL)
+static inline int is_configured_as_tls1_3(const mbedtls_ssl_config *conf)
```

Comment:
Suggestion: Rename to `ssl_conf_is_tls13_only()`.

Reply:
Done in #337